docs/visualization/path/response_time.md

```diff
@@ -45,113 +45,113 @@ plot.show()
 plot.show()
 ```
 
-![response_time_worst_in_input_histogram](../imgs/response_time_histogram_worst_in_input.png)
+![response_time_histogram_worst_with_external_latency](../imgs/response_time_histogram_worst_with_external_latency.png)
 
 ## Stacked Bar
 
 The following scripts generate stacked bar graphs of response time.
-Horizontal axis of the stacked bar graph means timeseries and vertical axis means `Response time` in the worst, best, all, worst-in-input case.
+Horizontal axis of the stacked bar graph means timeseries and vertical axis means `Response time` in the all, best, worst, worst-with-external-latency case.
 
 ```python
-# plot worst case
+# plot all case
 plot = Plot.create_response_time_stacked_bar_plot(path)
 plot.show()
 # or
-# plot = Plot.create_response_time_stacked_bar_plot(path, case='worst')
+# plot = Plot.create_response_time_stacked_bar_plot(path, case='all')
 # plot.show()
 ```
 
-![response_time_worst_stacked_bar](../imgs/response_time_stacked_bar_worst.png)
+![response_time_stacked_bar_all](../imgs/response_time_stacked_bar_all.png)
 
 ```python
 # plot best case
 plot = Plot.create_response_time_stacked_bar_plot(path, case='best')
 plot.show()
 ```
 
-![response_time_best_stacked_bar](../imgs/response_time_stacked_bar_best.png)
+![response_time_stacked_bar_best](../imgs/response_time_stacked_bar_best.png)
 
 ```python
-# plot worst-in-input case
-plot = Plot.create_response_time_stacked_bar_plot(path, case='worst-in-input')
+# plot worst case
+plot = Plot.create_response_time_stacked_bar_plot(path, case='worst')
 plot.show()
 ```
 
-![response_time_worst_in_input_stacked_bar](../imgs/response_time_stacked_bar_worst_in_input.png)
+![response_time_stacked_bar_worst](../imgs/response_time_stacked_bar_worst.png)
 
 ```python
-# plot all case
-plot = Plot.create_response_time_stacked_bar_plot(path, case='all')
+# plot worst-with-external-latency case
+plot = Plot.create_response_time_stacked_bar_plot(path, case='worst-with-external-latency')
 plot.show()
 ```
 
-![response_time_all_stacked_bar](../imgs/response_time_stacked_bar_all.png)
+![response_time_stacked_bar_worst_with_external_latency](../imgs/response_time_stacked_bar_worst_with_external_latency.png)
 
 The horizontal axis can be changed to `system time` or `index` by changing `plot.show()` to `plot.show(xaxis_type='index')`.
 
 `[worst - best] response time` in legends refers to the difference in response time between the worst case and the best case.
 
 ## TimeSeries
 
-The following scripts generate timeseries graphs of response time. The horizontal axis means time, labeled as Time [s]. The vertical axis means `Response time` in the best case, worst case, worst-in-input case, or all case.
+The following scripts generate timeseries graphs of response time. The horizontal axis means time, labeled as Time [s]. The vertical axis means `Response time` in the all, best, worst, worst-with-external-latency case.
```

Comment:
```suggestion
The following scripts generate timeseries graphs of response time. The horizontal axis means `system time` or `index`. The vertical axis means `Response time`.
```

Reply:
Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).